src/routes/blog/post/announcing-time-helper-queries/+page.markdoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Announcing time helper queries: Cleaner, more expressive time-based queries"
+description: Handle time-based filtering directly in your queries using new helpers for created and updated timestamps.
+date: 2025-09-02
+cover: /images/blog/announcing-time-helper-queries/cover.png
+timeToRead: 5
+author: jake-barnby
+category: announcement
+featured: false
+---
+If you’ve ever built a feed, a dashboard, or an audit report, you know how often you need to slice data by time. “Show me posts from last week,” “Export everything updated after the last backup,” or “Delete anything created before a certain retention window.”
+
+These time-based queries show up in so many of the apps you build, and when you’re working with dates, you naturally think in terms of *before* and *after*.
+
+To make this workflow smoother, we’re excited to announce **time helper queries:** A new way to filter your documents by creating and updating times using simple, expressive syntax.
+
+# A better way to query by dates
+
+Many workflows depend on knowing when a document was created or last updated. Whether you’re loading new items into a feed, exporting data that has changed since the previous run, or applying retention rules, you often need to filter by time.
+
+Previously, this meant comparing against `$createdAt` or `$updatedAt` using range operators. That approach works, but it adds extra noise to your queries. To make this more direct, you can now use dedicated helpers that map exactly to these common cases:
+
+- `createdBefore`: Find rows created before a given date
+- `createdAfter`: Find rows created after a given date
+- `updatedBefore`: Find rows updated before a given date
+- `updatedAfter`: Find rows updated after a given date
+
+```server-nodejs
+import { Client, Query, TablesDB } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const tablesDB = new TablesDB(client);
+
+tablesDB.listRows(
```

Comment:
We should use the new object params here. Looks like we haven't updated the Rows doc yet due to clashing PRs